snap/snapcraft.yaml

```diff
@@ -295,28 +209,46 @@ apps:
+      IGN_SENSORS_PLUGIN_PATH: '$SNAP/opt/ros/snap/lib'
+      GZ_SENSORS_PLUGIN_PATH: '$SNAP/opt/ros/snap/lib'
+      IGN_GUI_PLUGIN_PATH: $SNAP/opt/ros/snap/lib/ign-gazebo-6/plugins/gui:$SNAP/opt/ros/snap/lib/ign-gazebo-6/plugins:$SNAP/opt/ros/snap/lib/ign-gui-6/plugins'
+      LD_LIBRARY_PATH: '$SNAP/usr/local/lib/:$SNAP/usr/local/lib/$SNAPCRAFT_ARCH_TRIPLET/:$SNAP/usr/lib/:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/:$SNAP/usr/local/lib/site_ruby/3.0.0:$SNAP/usr/local/lib/$SNAPCRAFT_ARCH_TRIPLET/site_ruby:$SNAP/usr/local/lib/site_ruby:$SNAP/usr/lib/ruby/vendor_ruby/3.0.0:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/ruby/vendor_ruby/3.0.0:$SNAP/usr/lib/ruby/vendor_ruby:$SNAP/usr/lib/ruby/3.0.0:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/ruby/3.0.0:${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}'
+      IGN_GAZEBO_RENDER_ENGINE_PATH: '$SNAP/opt/ros/snap/lib/ign-rendering-6/engine-plugins/'
+      GZ_SIM_RENDER_ENGINE_PATH: '$SNAP/opt/ros/snap/lib/ign-rendering-6/engine-plugins/'
     command: usr/bin/ruby $SNAP/opt/ros/snap/bin/ign
-    plugs: [network, network-bind, home]
-    extensions: [kde-neon, ros2-foxy]
+    plugs:
+    - network
+    - network-bind
+    - home
+    extensions: [ros2-humble, kde-neon]
 
   ros-gz-create:
-    command: opt/ros/foxy/bin/ros2 run ros_ign_gazebo create
+    command: ros2 run roz_gz_sim create
     environment:
-        FASTRTPS_DEFAULT_PROFILES_FILE: ${SNAP}/usr/share/fastdds_no_shared_memory.xml
+      FASTRTPS_DEFAULT_PROFILES_FILE: ${SNAP}/usr/share/fastdds_no_shared_memory.xml
+      LD_LIBRARY_PATH: '$SNAP/usr/local/lib/:$SNAP/usr/local/lib/$SNAPCRAFT_ARCH_TRIPLET/:$SNAP/usr/lib/:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/:$SNAP/usr/local/lib/site_ruby/3.0.0:$SNAP/usr/local/lib/$SNAPCRAFT_ARCH_TRIPLET/site_ruby:$SNAP/usr/local/lib/site_ruby:$SNAP/usr/lib/ruby/vendor_ruby/3.0.0:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/ruby/vendor_ruby/3.0.0:$SNAP/usr/lib/ruby/vendor_ruby:$SNAP/usr/lib/ruby/3.0.0:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/ruby/3.0.0:${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}'
     plugs: [network, network-bind, home]
-    extensions: [ros2-foxy]
+    extensions: [ros2-humble, kde-neon]
 
   ros-gz-bridge:
-    command: opt/ros/foxy/bin/ros2 run ros_ign_bridge
+    command: ros2 run ros_gz_bridge
     environment:
-        FASTRTPS_DEFAULT_PROFILES_FILE: ${SNAP}/usr/share/fastdds_no_shared_memory.xml
+      FASTRTPS_DEFAULT_PROFILES_FILE: ${SNAP}/usr/share/fastdds_no_shared_memory.xml
+      LD_LIBRARY_PATH: '$SNAP/usr/local/lib/:$SNAP/usr/local/lib/$SNAPCRAFT_ARCH_TRIPLET/:$SNAP/usr/lib/:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/:$SNAP/usr/local/lib/site_ruby/3.0.0:$SNAP/usr/local/lib/$SNAPCRAFT_ARCH_TRIPLET/site_ruby:$SNAP/usr/local/lib/site_ruby:$SNAP/usr/lib/ruby/vendor_ruby/3.0.0:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/ruby/vendor_ruby/3.0.0:$SNAP/usr/lib/ruby/vendor_ruby:$SNAP/usr/lib/ruby/3.0.0:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/ruby/3.0.0:${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}'
```

Comment:
This could be set in a command-chain to make it cleaner

Reply:
done